firefox-ios/Storage/DefaultSuggestedSites.swift

```diff
@@ -129,8 +132,31 @@ open class DefaultSuggestedSites {
             )
         ]
     ]
+     */
+
+    public static let sites = [
+            SuggestedSite(
+                url: Ecosia.Environment.current.urlProvider.financialReports.absoluteString,
+                title: NSLocalizedString("Financial reports", tableName: "Ecosia", comment: ""),
+                trackingId: 901,
+                faviconResource: .bundleAsset(name: "financialReports", forRemoteResource: URL(string: "asset://defaultEcosiaFavicon")!)
+            ),
+            SuggestedSite(
+                url: Ecosia.Environment.current.urlProvider.privacy.absoluteString,
+                title: NSLocalizedString("Privacy", tableName: "Ecosia", comment: ""),
+                trackingId: 902,
+                faviconResource: .bundleAsset(name: "privacy", forRemoteResource: URL(string: "asset://defaultEcosiaFavicon")!)
+            ),
+            SuggestedSite(
+                url: Ecosia.Environment.current.urlProvider.trees.absoluteString,
+                title: NSLocalizedString("Trees update", tableName: "Ecosia", comment: ""),
+                trackingId: 903,
+                faviconResource: .bundleAsset(name: "treesUpdate", forRemoteResource: URL(string: "asset://defaultEcosiaFavicon")!)
+            )
+    ]
 
     public static func defaultSites() -> [Site] {
+        /* Ecosia: Simpli return Ecosia sites
         let locale = Locale.current
         let defaultSites = sites[locale.identifier] ?? sites["default"]
         return defaultSites?.map { data in
```

Comment:
While reviewing leftover files, I realised that this `defaultSites` function is equivalent to what we used to do in `SuggestedSitesCursor` ([these lines](https://github.com/ecosia/ios-browser/blob/main/Storage/SuggestedSites.swift#L25-L43)), which is then used in `TopSitesProvider`.

I don't fully understand what's the difference from that to simply returning the sites here, but did you test whether they result the same?

Btw, will remove the leftover files (`DefaultSuggestedSites` and `SuggestedSite`) on the old `Storage` folder since you've handled reintroducing their changes here 🙏